consensus/src/chained_bft/test_utils/mod.rs

```diff
@@ -78,8 +127,27 @@ impl TreeInserter {
             parent.round(),
             parent.quorum_cert().certified_block_id(),
             parent.quorum_cert().certified_block_round(),
+            false,
         );
+        self.insert_block_with_qc(parent_qc, parent, round)
+    }
 
+    // Insert a block that is guaranteed to be committed.
+    // This function is only to be used for testing other components, given that the block will commit.
+    pub fn insert_guaranteed_block(
```

Comment:
we probably can use this as default and delete `insert_block` unless there're tests failures